vars/perfSearchBestConfigParallelPipeline.groovy

```diff
@@ -77,7 +77,10 @@ def call(Map pipelineParams) {
             string(defaultValue: '', description: 'Number of stress process per loader', name: 'n_stress_process')
             string(defaultValue: '', description: 'Number of process add/remove on each round', name: 'stress_process_step')
 
-            string(defaultValue: '', description: 'Relative difference between current and best to choose new best result', name: 'max_deviation')
+            string(defaultValue: '', description: 'Stress step duration for c-s command, ex: 15m, 3h, etc. If not set step duration is 15 min',
```

Comment:
@rayakurl empty value here will cause not setting the appropriate env var. In this case the default value from the `defaults/test_default.yaml` file will be used.
If it gets set to anything different than empty value by default then the value from the `defaults/test_default.yaml` file will always be ignored. And it doesn't sound ok.

Reply:
`s/"If not set step duration is 15 min"/"If not set, will take default from test_default.yaml"`

Reply:
test_default.yaml has this parameter set as "", But test configured with 15m in test_config yaml 'test-cases/performance/perf-search-best-throughput-config.yaml`. if parameter is not set in pipeline, then value would be taken from `test-cases/performance/perf-search-best-throughput-config.yaml`. The main config yaml for test is: `test-cases/performance/perf-search-best-throughput-config.yaml` and these 2 new parameters just gives a bit more flexibility, do not change main config yaml, create a PR and wait merge and backport.
i changed the string.

Reply:
> > `s/"If not set step duration is 15 min"/"If not set, will take default from test_default.yaml"`
> 
> Default value configured in test config yaml, not test_default.yaml. So Raya's suggestions is better. @fgelcer WDYT?

it is in `test_default.yaml`, in the same commit...
see here --> https://github.com/scylladb/scylla-cluster-tests/blob/70603c1e9398cab21b18668dd03faf0f418b8db2/defaults/test_default.yaml#L225

Reply:
but i don't really care where it is, just that, if we decide to change the default. we will never remember to change this parameter description here, and it will be just wrong